global/scripts/CorrThick.sh

```diff
@@ -0,0 +1,171 @@
+opts_AddOptional '--regnames' 'RegNamesStr' 'my reg str' "set the desired registration name(s) separated by @, 'string' 'RegName@RegName@RegName@...etc.' default MSMSulc" "MSMSulc"
+opts_AddOptional '--hemi' 'Hemi' 'hemisphere' "provide hemisphere for regression calculation, L=Left, R=Right, or B=Both, default 'B'" "B"
+opts_AddOptional '--surf' 'Surface' 'surface' "provide surface for regression calculation, white or midthickness, default midthickness" "midthickness"
+opts_AddOptional '--patch-size' 'PatchSize' 'distance' "provide patch kernel size in millimeters FWHM for regression, default 6" "6"
+opts_AddOptional '--surf-smooth' 'SurfSmooth' 'distance' "provide surface smoothing in millimeters FWHM, default 2.14" "2.14"
+opts_AddOptional '--metric-smooth' 'MetricSmooth' 'distance' "provide metric smoothing in millimeters FWHM, default 2.52" "2.52"
+opts_AddOptional '--skip-computation' 'SkipCompute' 'YES or NO' "whether or not to compute the curvature-corrected (folding-compensated) cortical thickness, if it is already available, but just to resample it to 164k and 32k, defaults to 'NO'" "NO"
+
+# The papers associated with this pipeline are available at: 
+
+# https://www.biorxiv.org/content/10.1101/2025.05.03.651968v1 and
+# https://onlinelibrary.wiley.com/doi/full/10.1002/hbm.25776
+
+# Please cite these work and give proper attribution when using this measure.
+
+# This pipeline requires several Python dependencies: numpy, nibabel, scipy, psutil (plus standard libraries: os, math, multiprocessing, concurrent.futures) 
+
+# To run this pipeline stand-alone (without rerunning the full HCP pipeline), 
+# prepare a batch script that defines: 
+# - Your subject directory 
+# - The subject list 
+# - Registration method(s) (MSMSulc, MSMAll, or both) 
+# - The location of the HCP Pipeline environment setup script (SetUpHCPPipeline.sh) 
+
+# Example batch script: 
+
+####################################### 
+#EnvironmentScript="/path/to/SetUpHCPPipeline.sh" # HCP Pipeline environment script 
+#source "$EnvironmentScript" 
+
+#SubjectDir="/path/to/subjects" 
+#SubjList="100206 100307 100408 ...." # space-separated subject IDs 
+
+#RegName="MSMSulc@MSMAll" # one or more registrations 
+
+#for Subject in $SubjList ; do 
+	#bash "$HCPPIPEDIR/global/scripts/CorrThick.sh" --subject-dir="$SubjectDir" --subject="$Subject" --regnames="$RegNames" --skip-computation=NO 
+#done 
+#######################################
+
+opts_ParseArguments "$@"
+
+if ((pipedirguessed))
+then
+	log_Err_Abort "HCPPIPEDIR is not set, you must first source your edited copy of Examples/Scripts/SetUpHCPPipeline.sh"
+fi
+
+#display the parsed/default values
+opts_ShowValues
+
+#sanity check boolean strings and convert to 1 and 0
+SkC=$(opts_StringToBool "$SkipCompute")
+
+#set paths
+NonlinearFolder="$SubjectDir"/"$Subject"/MNINonLinear
+NativeFolder="$NonlinearFolder"/Native
+T1wFolder="$SubjectDir"/"$Subject"/T1w/Native
```

Comment:
`$T1wFolder` containing "/Native" might be confusing (and possibly cause bugs if code is copy-pasted between this and a different pipeline):

https://github.com/Washington-University/HCPpipelines/blob/45d553215e0288fec620e8cb7b802bca6bb190ee/PreFreeSurfer/PreFreeSurferPipeline.sh#L357

I suggest renaming this variable to something like `$T1wNativeFolder`.